src/main/java/com/diffblue/deeptestutils/Reflector.java

```diff
@@ -329,14 +330,12 @@ private static void makePublic(final CtClass c) {
    * private default constructor, it is made accessible and is then called. Else
    * we use Objenesis to force creation of an instance.
    *
-   * There are several exceptions that `Constructor.netInstance` can throw. We
+   * There are several exceptions that `Constructor.newInstance` can throw. We
```

Comment:
Should ideally be in the previous commit rather than introducing a typo in one commit and fixing it in the next.